### PR TITLE
feat(sns-cli): Rename Distribution.InitialBalances.governance as Distribution.InitialBalances.treasury

### PR DESCRIPTION
**Pull request by aterga**
This PR adjusts the SNS configuration file after `Distribution.InitialBalances.governance` has been remapped from 
`initialTokenDistribution.treasuryDistribution.total` to
`Distribution.InitialBalances.treasury` in https://github.com/dfinity/ic/pull/1755.

### CODE REVIEW

**Comment by aterga** [comment]
Merge this PR together with: 
* https://github.com/dfinity/sns-testing/pull/81/files
* https://github.com/dfinity/ic/pull/1755

File: bin/versions.bash [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # shellcheck disable=SC2034 # Variables are expected to be unused in this file
 # Release from 2024-05-02 which includes nervous_system_parameters.
 SNS_AGGREGATOR_RELEASE=proposal-129614-agg
-DFX_IC_COMMIT=36cbd803b7f6e5f811c720361e648b614ef66162
+DFX_IC_COMMIT=a1a9e39d6e94a48cc3f2a461cf328032b37b8b21
```
Comment (aterga) [inline bin/versions.bash:5]:
```suggestion
DFX_IC_COMMIT=2b3b69ee4bdad75776a0ed668aa53500600af044
```

**Comment by dskloetd** [comment]
What does it mean to merge PRs from different repos together?

**Comment by aterga** [comment]
> What does it mean to merge PRs from different repos together?

I removed that memo to avoid confusion.

**Comment by dskloetd** [comment]
Can you link the actual PR that made the change from the PR description?

**Review by daniel-wong-dfinity-org — APPROVED** [review]

**Pull request merged**